tests/components/litejet/test_config_flow.py

```diff
@@ -0,0 +1,77 @@
+"""The tests for the litejet component."""
+from unittest.mock import patch
+
+from serial import SerialException
+
+from homeassistant.components.litejet.const import DOMAIN
+from homeassistant.const import CONF_PORT
+
+from tests.common import MockConfigEntry
+
+
+async def test_show_config_form(hass):
+    """Test show configuration form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": "user"}
+    )
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "user"
+
+
+async def test_create_entry(hass, mock_litejet):
+    """Test create entry from user input."""
+    test_data = {CONF_PORT: "/dev/test"}
+
+    result = await hass.config_entries.flow.async_init(
```

Comment:
We should patch the integration setup functions on all create entry results to avoid setting up the whole integration.